us/en/skymiles/blocks/toc/toc.js

```diff
@@ -0,0 +1,16 @@
+export default function decorate(block) {
+  const headings = document.querySelectorAll('h3');
+  const toc = document.createElement('ul');
+
+  headings.forEach((heading) => {
+    const item = document.createElement('li');
+    const link = document.createElement('a');
+
+    link.href = `#${heading.id}`;
+    link.textContent = heading.textContent;
```

Comment:
If we have a styled heading, it might be better to leverage `innerHTML` for both